### PR TITLE
is_striving_for_api_consistency

### PR DESCRIPTION
**Pull request by mzieg**
I don't like adding weight to obvious 1-line convenience accessors but different methods following the same naming style are returning different types.

### CODE REVIEW

File: wasatch/FeatureIdentificationDevice.py [hunk 1]
```diff
@@ -1978,6 +1975,8 @@ def is_laser_firing(self) -> SpectrometerResponse:
         """
```
Comment (mzieg) [inline wasatch/FeatureIdentificationDevice.py:1975]: This is an exception to the rule.

**Comment by mzieg** [comment]
I would be fine deleting all the "-> bool" when there are no exceptions :-)  Maybe we add a nomenclature to signify methods returning SpectrometerResponse?  The class itself was a good idea.

**Comment by mzieg** [comment]
Note that FID.can_laser_fire() technically falls under the "has_" and "is_" nomenclature too (returns SpectrometerResponse, so technically another exception, but I understand why that was done).

**Review by samiebee43 — CHANGES_REQUESTED** [review]
https://realpython.com/lessons/type-comments/

File: wasatch/DetectorRegions.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def remove(self, roi):
     def count(self):
         return len(self.regions)
 
-    def has_region(self, region):
+    def has_region(self, region) -> bool:
```
[the remark this answers is not in the document]
Comment (mzieg) [reply wasatch/DetectorRegions.py]: fixed

**Review by samiebee43 — APPROVED** [review]

**Pull request merged**